src/nci_transition_idle_to_discovery.c

```diff
@@ -1231,7 +1278,11 @@ nci_transition_idle_to_discovery_get_config_rsp(
             data.size = payload->size - 2;
             if (cmd_status == NCI_STATUS_OK) {
                 GDEBUG("%c CORE_GET_CONFIG_RSP ok", DIR_IN);
-                if (nci_transition_idle_to_discovery_la_nfcid1_ok(n,
+                 if (nci_transition_idle_to_discovery_la_sens_res_1_ok(n,
+                    &data, &la_sens_res_1)) {
+                    set_config &= ~CORE_SET_CONFIG_LA_SENS_RES_1;
+                }
+               if (nci_transition_idle_to_discovery_la_nfcid1_ok(n,
```

Comment:
This line and also the if above appears to have indentation mismatches.

Reply:
indeed!